awx/main/wsrelay.py

```diff
@@ -310,19 +310,30 @@ async def run(self):
         for k, v in settings.LISTENER_DATABASES.get('default', {}).get('OPTIONS', {}).items():
             database_conf['OPTIONS'][k] = v
 
-        async_conn = await psycopg.AsyncConnection.connect(
-            dbname=database_conf['NAME'],
-            host=database_conf['HOST'],
-            user=database_conf['USER'],
-            password=database_conf['PASSWORD'],
-            port=database_conf['PORT'],
-            **database_conf.get("OPTIONS", {}),
-        )
+        try:
+            async_conn = await psycopg.AsyncConnection.connect(
+                dbname=database_conf['NAME'],
+                host=database_conf['HOST'],
+                user=database_conf['USER'],
+                password=database_conf['PASSWORD'],
+                port=database_conf['PORT'],
+                **database_conf.get("OPTIONS", {}),
+            )
+        except Exception as e:
+            logger.error(e)
```

Comment:
I agree with not showing the full stack trace here, but I want the log to identify itself very specifically like

```python
logger.error(f'Failed to establish wsrelay async connection {e}')
```

You can use the python logging format stuff instead of f-string if you like

And I want _every_ log message to have a different string. No two log messages in different parts of the code should be the same.